tests/testthat/test-probabilities.R

```diff
@@ -44,9 +44,5 @@ test_that("Probabilities", {
     for (input in valid.input) {
         expect_silent(validateProbabilityArguments(input, NULL))
         expect_silent(validateProbabilityArguments(input, data.frame(x = 1)))
-        # Default method will error as the methods are provided in flipTrees,
```

Comment:
Why are we removing this?

Reply:
It was a check that for when they call `Probabilities` when `flipRegression` is not loaded (so the `Probabilities.Regression` method isn't available. It gives a nice error message but has little utility I believe (no client on Displayr/Q would experience it and only open source users). The tests are blocking the CI from passing and given I'm a one man band right now I thought best to remove them.